modules/graphql_views/src/Plugin/Deriver/ViewDeriver.php

```diff
@@ -113,6 +113,7 @@ public function getDerivativeDefinitions($basePluginDefinition) {
         $paged = TRUE;
         $arguments += [
           'page' => ['type' => 'Int', 'default' => $this->getPagerOffset($display)],
+          'offset' => ['type' => 'Int', 'default' => $this->getPagerOffset($display)],
```

Comment:
Why is the default value of the 'page' set to $this->getPagerOffset($display)? Isn't that returning the offset?